README.md

```diff
@@ -137,20 +131,28 @@ any value used for that key will be overridden by the child value. _With
 collisions the child always overrides the parent_
 
 ```ts
-class Test1 extends Serializable {
-  @SerializeProperty("serialize_me")
-  serializeMe = "nice1";
-}
+import { Serializable, SerializeProperty } from "./mod.ts";
+import { assertEquals, test } from "./test_deps.ts";
+
+test({
```

Comment:
I think it may be a bit more readable to not use tests in the readme. I did some testing with deno assertions and I think that rather than relying on `deno test` I now think it may be better to just run the sample code and check the processes response code.

I will make a discussion/issue for running the example code,  I think going so far as to rewrite all the example code is out of scope for this ticket. If you want to keep the `test` blocks though I think that's fine for now too.

Reply:
no hard thoughts on it here, want to keep it for now and we can decide when we add the other script to run the code blocks?